requirements.txt

```diff
@@ -1,5 +1,6 @@
-qiskit>=0.25
-pennylane>=0.20
+qiskit>=0.32
+mthree>=0.17.2
```

Comment:
How come this was required, does it not get shipped with Qiskit? Should it be added to `setup.py` too?